index.d.ts

```diff
@@ -0,0 +1,36 @@
+import { Model, ModelClass, Page } from 'objection';
+import { QueryBuilder } from 'knex';
+
+declare module 'objection-hashid' {
+  class AuthQueryBuilder<M extends Model, R = M[]> extends Model.QueryBuilder<M, R> {
+    ArrayQueryBuilderType: AuthQueryBuilder<M, M[]>;
+    SingleQueryBuilderType: AuthQueryBuilder<M, M>;
+    NumberQueryBuilderType: AuthQueryBuilder<M, number>;
+    PageQueryBuilderType: AuthQueryBuilder<M, Page<M>>;
+  }
+
+  class HashIdModelClass extends Model {
+    QueryBuilderType: AuthQueryBuilder<this>;
+    QueryBuilder: AuthQueryBuilder<this>;
+
+    static get hashIdSalt(): string;
+
+    static get hashIdMinLength(): number | void;
+
+    static get hashIdAlphabet(): string | void;
+
+    static get hashIdSeps(): string | void;
+
+    static get _hashIdInstance(): string;
```

Comment:
So the type itself is wrong, as it's supposed to be an instance of `hashids`, but it doesn't matter anyway since this is supposed to be a private property(!)

Reply:
For future reference, when a method/property starts with an underscore, it's meant to be _private_ and the outside world _shouldn't_ even know that it exists(!)

Reply:
Whoops, thanks for letting me know.